members/views/AccountCreate.py

```diff
@@ -0,0 +1,105 @@
+from django.urls import reverse
+from django.views.decorators.clickjacking import xframe_options_exempt
+from members.forms import signupForm
+from django.shortcuts import render
+from django.utils import timezone
+from django.http import HttpResponseRedirect
+from django.contrib.auth.models import User
+from members.models.family import Family
+from members.models.person import Person
+
+
+@xframe_options_exempt
+def AccountCreate(request):
+    if request.method == "POST":
+        # figure out which form was filled out.
+        if request.POST["form_id"] == "signup":
+            # signup has been filled
+            signup = signupForm(request.POST)
+            if signup.is_valid():
+                # check if family already exists
+                # TODO: rewrite this! >>>>
+                try:
```

Comment:
I stedet for try catch kan du overveje en funktion som https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#get-object-or-404

Reply:
Igen, det er ikke ny kode. Det er bare eksisterende kode som er flyttet.